scripts/booklist-classes.js

```diff
@@ -0,0 +1,57 @@
+/* eslint max-classes-per-file: ["error", 2] */
+class Book {
+  constructor() {
+    this.title = document.getElementById('book-title-input').value;
+    this.author = document.getElementById('book-author-input').value;
+  }
+}
+
+class Bookshelf {
+  constructor() {
+    this.books = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+    this.bookList = document.getElementById('book-list');
+  }
+
+  removeBook(index) {
+    this.books = this.books.filter((book) => this.books.indexOf(book) !== index);
+    this.storeData();
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] [Optional] This issue is easy to fix it instead of disabling eslint rule, would you try harder to fix it? (Fixing eslint issues will increase your code quality and code readability)